src/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+</head>
+<body>
+    <h1>Leaderboard</h1>
+    <section id="container">
+        <div id="left-column">
+            <div id="recent-scores-title">
+                <h2>Recent scores</h2>
+                <button>Refresh</button>
+            </div>
+                <ul id="recent-scores">
+                    <li>name: 100</li>
+                    <li>name: 100</li>
```

Comment:
- Hey, although this wasn't mentioned in the project requirement, do have in mind that this section should be populated dynamically in the JS file. please, kindly update this.